README.md

```diff
@@ -47,6 +47,14 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag's Configuration
+
+The connector reads from the tag's historical logs to write to OSIsoft. For best results, enable the Logging Deadband feature. The settings for the historical log deadband are the same as the settings for the OSIsoft connector deadband.
```

Comment:
"The settings for the historical log deadband are the same as the settings for the OSIsoft connector deadband." - what do you mean by this, the osisoft connector doesnt have a deadband setting. Did you mean the historical log deadband setting will be shared with the OSIsoft connector?

Reply:
I was trying to convey the fact that the historical logs are read into the connector in a clear way.. Probably did the opposite of that. I think I'll just say exactly that - "The historical logs on the Flexy are read into the connector." And I'll change the rest to just tell them to pay attention to the settings for historical logging

Reply:
I am not sure if stating just that will be clear, you may want to incorporate that the historical logs are the source for the data pushed to osisoft.